src/Commands/InitCommand.php

```diff
@@ -90,6 +92,15 @@ public function handle(): void
 
         $this->appUrl = $this->ask('Please enter an application URL', "https://api.dev.{$kebabName}.com");
 
+        if (!file_exists(filename: '.env.example')) {
```

Comment:
@artengin not sure we need to make it as:
- you've written that the `.env.example` always exists after the fresh Laravel installation
- we have to support our `.env.example` file and actualize it

Let's remove this part for now